content/docs/help/faq.md

```diff
@@ -48,6 +48,26 @@ Of course biscuit is a recent piece of tech, which makes it harder to justify th
 
 Consider unionizing to protect yourself from being unduly fired.
 
+## How can biscuit be integrated with OAuth and OIDC?
+
+Biscuit is focused on the authorization and delegation side of API access, so it can be used as an OAuth access token. OAuth does not mandate a specific format for the access token.
+Refresh tokens work better when they are meant for a single use. In that case, a random string stored in the Authorization Server (in OAuth terms) database will fit the task.
+OIDC specifies that identity information is transmitted using JWTs, so Biscuit will not be usable in their place.
+
+## Why Datalog?
+
+Biscuit needed a language that could be serialized in a compact format, and executed in the same way across different implementations in different languages. It also had to be fast to execute and have a limited impact on performance. And it had to be general enough, so it would not prescribe a specific authorization model.
```

Comment:
```suggestion
Biscuit needed a language that could be serialized in a compact format, and executed in the same way across different implementations in different languages. It also had to be fast to execute and have a limited impact on memory use. Finally, it had to be general enough, so it would not prescribe a specific authorization model.
```

Reply:
as for performance, speed is mentionned just before, so it reads a bit strange to have both speed and performance mentioned separately.